src/Aggregation/Reduced.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ElegantBro\Arrayee\Aggregation;
+
+use ElegantBro\Arrayee\Just;
+use ElegantBro\Interfaces\Arrayee;
+use Exception;
+use function array_reduce;
+
+final class Reduced implements Arrayee
+{
+    /**
+     * @var Arrayee
+     */
+    private $arrayee;
+
+    /**
+     * @var callback
+     */
+    private $callback;
+
+    /**
+     * @var Arrayee
+     */
+    private $initial;
+
+    /**
+     * Reduced constructor.
+     * @param Arrayee $arrayee
+     * @param callable(array, mixed): array $callback function(array $carry, $item): array
```

Comment:
`callable(array, mixed): array` is not valid annotation type.

`@param callable $callback Function that reduces items to array <code>function(array $carry, $item): array</code>`